lib/internal/event_target.js

```diff
@@ -58,6 +58,7 @@ const kWeakHandler = Symbol('kWeak');
 const kResistStopPropagation = Symbol('kResistStopPropagation');
 
 const kHybridDispatch = SymbolFor('nodejs.internal.kHybridDispatch');
+const kRemoveWeakListenerHelper = SymbolFor('nodejs.internal.removeWeakListenerHelper');
```

Comment:
Why `SymbolFor` and not `Symbol`?

Reply:
Just used like `kHybridDispatch`

Reply:
updated

Reply:
> Just used like kHybridDispatch

I assumed so, but it basically means it could be reached from outside by calling `Symbol.for('....')` from user land - which I don't think we want here 🙂

Reply:
yeah this is why I changed :)